packages/lib/CalendarService.ts

```diff
@@ -162,6 +162,7 @@ export default abstract class BaseCalendarService implements Calendar {
          * [UPDATE]: Since we're not using the PUBLISH method to publish the iCalendar event and creating the event directly on iCal,
          * this shouldn't be an issue and we should be able to add attendees to the event right here.
          */
+        ...(calEventRaw.hideCalendarEventDetails ? { classification: "PRIVATE" } : {}),
```

Comment:
this is throwing an error
![Screenshot 2024-09-26 at 11 18 22 AM](https://github.com/user-attachments/assets/5405c60f-ca25-410e-bff0-443bb524a249)